src/SignalR/SignalR/Cmdlets/NewAzureRmSignalR.cs

```diff
@@ -53,88 +55,86 @@ public sealed class NewAzureRmSignalR : SignalRCmdletBase
 
         [Parameter(
             Mandatory = false,
-            HelpMessage = "The SignalR service SKU.")]
+            HelpMessage = "The SignalR service SKU. Default to \"Standard_S1\".")]
         [PSArgumentCompleter("Free_F1", "Standard_S1")]
-        public string Sku { get; set; } = DefaultSku;
+        public string Sku { get; set; }
 
         [Parameter(
             Mandatory = false,
-            HelpMessage = "The SignalR service unit count, from 1 to 10. Default to 1.")]
-        [PSArgumentCompleter("1", "2", "3", "4", "5", "6", "7", "8", "9", "10")]
-        [ValidateRange(1, 10)]
-        public int UnitCount { get; set; } = 1;
+            HelpMessage = "The SignalR service unit count, value only from {1, 2, 5, 10, 20, 50, 100}. Default to 1.")]
+        [PSArgumentCompleter("1", "2", "5", "10", "20", "50", "100")]
+        public int UnitCount { get; set; } = DefaultUnitCount;
 
         [Parameter(
             Mandatory = false,
             HelpMessage = "The tags for the SignalR service.")]
         public IDictionary<string, string> Tag { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "The service mode for the SignalR service.")]
+        [PSArgumentCompleter("Default", "Serverless", "Classic")]
+        public string ServiceMode { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "The allowed origins for the SignalR service. To allow all, use \"*\" and remove all other origins from the list. Slashes are not allowed as part of domain or after TLD")]
```

Comment:
Can we use avoid using abbr? Won't want our customers to guess what TLD means...

Reply:
@isra-fel

fixed